package.json

```diff
@@ -11,7 +11,7 @@
     "package-windows": "electron-builder --win",
     "build-headless": "tsc",
     "dist": "electron-builder",
-    "mac-publish": "export ELECTRON_BUILDER_ALLOW_UNRESOLVED_DEPENDENCIES=true && electron-builder --mac --win --linux -p always",
+    "build-packages-all-distributions": "export ELECTRON_BUILDER_ALLOW_UNRESOLVED_DEPENDENCIES=true && electron-builder --mac --win --linux -p always",
```

Comment:
Please change version to 1.8.39. I will publish a new minor shortly, so this may remain as a draft but still be helpful.

Reply:
what's the reason for increasing the version? there's zero impact on behavior, even on the ci.